### PR TITLE
Added icons to selected PS

### PR DESCRIPTION
**Pull request by mihai-macaneata**
/addresses #39

### CODE REVIEW

**Comment by xlotlu** [comment]
there are vector versions of the images in google drive.
you can convert them to svg from eps using `ps2pdf` from package `ghostscript`, and `pdf2svg`.

File: assets/js/components/Sectors.vue [hunk 1]
```diff
@@ -630,10 +630,10 @@ export default Vue.extend({
         // level 2 items are hidden
         .attr("opacity", (d) => d.depth == 2 ? 0 : null )
         // and really hidden
-        .style("display", (d) => d.depth == 2 ? "none" : null )
+        .style("display", (d) => d.depth == 2 ? "none" : null);
```
Comment (xlotlu) [inline assets/js/components/Sectors.vue]: the spacing there is meant to denote how the inner stuff is part of the function definition, and not another argument.

File: assets/js/components/Sectors.vue [hunk 2]
```diff
@@ -647,6 +647,17 @@ export default Vue.extend({
         .on('mouseover', this.tip.show)
         .on('mouseout', this.tip.hide);
 
+        aentered.append('use')
```
Comment (xlotlu) [inline assets/js/components/Sectors.vue]: it is simpler (and better) to do this in html, from vue

File: assets/js/components/mixins/WithSectors.js [hunk 3]
```diff
@@ -7,6 +7,23 @@ for (const sector of _SECTORS) {
...
 
+
+export function get_image(code) {
+  const sector = SECTORS[code];
+  if (!sector) throw "sector not found: " + code;
+  const sector_img = sector.name.toLowerCase().replace(/ /g, '');
+  return `${sector_img}`;
+}
+
+const req = require.context('svg-sprite-loader!imgs/fmIcons', false, /[a-z]+\.png$/);
```
Comment (xlotlu) [inline assets/js/components/mixins/WithSectors.js]: why are the sector icons called fmIcons?

File: assets/js/components/Sectors.vue (hunk 2 above)
Comment (mihai-macaneata) [inline assets/js/components/Sectors.vue]: Doing it in html would mean that the positioning of the icons would be absolute and the scaling would include a lot of media queries and hacky stuff

Comment (xlotlu) [inline assets/js/components/Sectors.vue]: how is `left: 50%; top: 50%; transform: translate(-50%,-50%); width: x%` a hack? no media queries needed. you just add to the `<chart-container>`, the svg is already absolutely positioned inside it.

**Comment by mihai-macaneata** [comment]
The icons are now HTML

**Comment by mihai-macaneata** [comment]
I chose to pre-render every image to avoid an image request every time you filter by a sector, by doing this I need one of the needless divs for the iteration. I will get rid of the other

**Comment by xlotlu** [comment]
> I chose to pre-render every image to avoid an image request every time 

that is bad design that needs to be fixed at the proper place: by using sprites. we can't allow separate images to make it into production.

you also happen to have a sprite readily generated. the svg one, rememeber?

**Pull request merged**